velox/type/CMakeLists.txt

```diff
@@ -16,6 +16,7 @@ if(${VELOX_BUILD_TESTING})
 endif()
 add_subdirectory(tz)
 
+add_compile_definitions(FOLLY_HAVE_INT128_T=1)
```

Comment:
@Yuhta Modifying the Folly build seems to be non-trivial. Can we work with this approach?

Reply:
Yeah should be ok.  Can we move this to project root though?

Reply:
Done. Moving to the project root was required actually.